Sources/Public/ItemViews/MonthGridBackgroundView.swift

```diff
@@ -0,0 +1,125 @@
+// Created by Bryan Keller on 1/30/23.
+// Copyright © 2023 Airbnb Inc. All rights reserved.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import UIKit
+
+// MARK: - MonthGridBackgroundView
+
+/// A background grid view that draws separator lines between all days in a month. 
+public final class MonthGridBackgroundView: UIView {
+
+  // MARK: Lifecycle
+
+  fileprivate init(invariantViewProperties: InvariantViewProperties) {
+    self.invariantViewProperties = invariantViewProperties
+    super.init(frame: .zero)
+    backgroundColor = .clear
+  }
+
+  required init?(coder _: NSCoder) {
+    fatalError("init(coder:) has not been implemented")
+  }
+
+  // MARK: Public
+
+  public override func draw(_: CGRect) {
+    let context = UIGraphicsGetCurrentContext()
+    context?.setLineWidth(invariantViewProperties.lineWidth)
+    context?.setStrokeColor(invariantViewProperties.color.cgColor)
+
+    if traitCollection.layoutDirection == .rightToLeft {
+      transform = .init(scaleX: -1, y: 1)
```

Comment:
Hmm, I haven't seen this done before I'm not sure if drawRect is a good place to change the transform. Could we just the origin.x of the rects instead? Also, I think we might need to call `setNeedsDisplay` in respond to `traitCollectionDidChange`.